src/colorizer/canvas/elements/scaleBar.ts

```diff
@@ -0,0 +1,130 @@
+import { Vector2 } from "three";
```

Comment:
Copied from `CanvasWithOverlay` with minimal changes to original logic. Main improvements are that text rendering is a little cleaner now and uses the `utils.ts` helper methods.